tests/sentry/tasks/test_post_process.py

```diff
@@ -1981,19 +1981,22 @@ def test_has_escalated(self, mock_run_post_process_job):
         event.group = Group.objects.get(id=group.id)
 
         with self.feature("projects:first-event-severity-new-escalation"):
-            self.call_post_process_group(
-                is_new=True,
-                is_regression=False,
-                is_new_group_environment=True,
-                event=event,
-            )
+            with patch("sentry.issues.issue_velocity.calculate_threshold", return_value=9999):
```

Comment:
didn't mock `get_latest_threshold` for this particular test so we could assure that the two locks work together + that redis runs okay in the job; all other tests this function was mocked in attempt to make the tests run faster but i'm open to change